aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -150,6 +152,7 @@ module aptos_framework::fungible_asset {
 		withdraw_function: Option<FunctionInfo>,
 		deposit_function: Option<FunctionInfo>,
         derived_balance_function: Option<FunctionInfo>,
+        derived_supply_function: Option<FunctionInfo>
```

Comment:
Oh wait... This is really a nasty issue. You wouldn't be able to add a new field to an existing struct. So you will need to put this function info in a new struct.